lib/test.js

```diff
@@ -22,6 +22,7 @@ function Test(title, fn) {
   Runnable.call(this, title, fn);
   this.pending = !fn;
   this.type = 'test';
+  this.body = (fn || '').toString();
```

Comment:
No - when the new test is created via `Test#clone`, it'll get the exact same string body https://github.com/danielstjules/mocha/blob/d59cc6c166d0d112a62016909b33e04ed08d2274/lib/test.js#L34